config/storage.yml

```diff
@@ -1,34 +1,27 @@
-test:
-  service: Disk
-  root: <%= Rails.root.join("tmp/storage") %>
-
 local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
-# Use bin/rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
-# amazon:
-#   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket-<%= Rails.env %>
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
 
-# Remember not to checkin your GCS keyfile to a repository
-# google:
-#   service: GCS
-#   project: your_project
-#   credentials: <%= Rails.root.join("path/to/gcs.keyfile") %>
-#   bucket: your_own_bucket-<%= Rails.env %>
+amazon:
+  service: S3
+  access_key_id: ENV["S3_ACCESS_KEY_ID"] %>
+  secret_access_key: ENV["S3_SECRET_ACCESS_KEY"] %>
+  region: us-east-1
+  bucket: your_own_bucket-<%= Rails.env %>
 
-# Use bin/rails credentials:edit to set the Azure Storage secret (as azure_storage:storage_access_key)
-# microsoft:
-#   service: AzureStorage
-#   storage_account_name: your_account_name
-#   storage_access_key: <%= Rails.application.credentials.dig(:azure_storage, :storage_access_key) %>
-#   container: your_container_name-<%= Rails.env %>
+# Replace credentials with your own GCS keyfile
+google:
+  service: GCS
+  project: your_project
+  credentials: <%= Rails.root.join("path/to/gcs.keyfile") %>
```

Comment:
We should probably set a more descriptive keyfile name such as `gcp-storage-keyfile.json` and then add that to the .gitignore here